sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c

```diff
@@ -118,7 +118,7 @@ struct FloatMat33 Ga_inv;
 struct FloatVect3 control_increment; // [dtheta, dphi, dthrust]
 
 float filter_cutoff = GUIDANCE_INDI_FILTER_CUTOFF;
-float guidance_indi_max_bank = GUIDANCE_H_MAX_BANK;
+float guidance_indi_max_bank = DegOfRad(GUIDANCE_H_MAX_BANK);
```

Comment:
I don't see why this should be converted to degrees, to convert it again to radians later

Reply:
The setting in the GCS is otherwise in radians.